src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -97,21 +100,46 @@ async def setup_pex_cli_process(
     pex_binary: PexBinary,
     pex_env: PexEnvironment,
     python_native_code: PythonNativeCode,
+    global_options: GlobalOptions,
 ) -> Process:
     tmpdir = ".tmp"
-    downloaded_pex_bin, tmp_dir_digest = await MultiGet(
+    gets: List[Get] = [
         Get(DownloadedExternalTool, ExternalToolRequest, pex_binary.get_request(Platform.current)),
         Get(Digest, CreateDigest([Directory(f"{tmpdir}/.reserve")])),
-    )
+    ]
+    cert_args = []
+
+    # The certs file will typically not be in the repo, so we can't digest it via a PathGlobs.
+    # Instead we manually create a FileContent for it.
+    if global_options.options.ca_certs_path:
+        ca_certs_content = Path(global_options.options.ca_certs_path).read_bytes()
+        chrooted_ca_certs_path = os.path.join(
+            os.path.basename(global_options.options.ca_certs_path)
+        )
```

Comment:
Did you mean to join this with something?

Reply:
Ooops, not any more